src/core/server/server.cc

```diff
@@ -1269,6 +1271,7 @@ grpc_error_handle Server::SetupTransport(Transport* transport,
     GRPC_TRACE_LOG(server_channel, INFO) << "Adding connection";
     connections_.emplace(std::move(t));
     ++connections_open_;
+    stream_quota_->IncrementOpenChannels();
   } else {
     CHECK(transport->filter_stack_transport() != nullptr);
```

Comment:
We'll need to do the increment/decrement for this branch also - the prior branch is callv3, this one is callv1, and we need both